e2e/src/main/java/gov/hhs/cdc/trustedintermediary/e2e/HapiParser.java

```diff
@@ -0,0 +1,28 @@
+package gov.hhs.cdc.trustedintermediary.e2e;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.fhirpath.IFhirPath;
+import ca.uhn.fhir.parser.IParser;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Base;
+import org.hl7.fhir.r4.model.Bundle;
+
+public class HapiParser {
```

Comment:
Java doc missing, also should this be wrapping an interface?

Reply:
This is a helper function to only use in e2e tests so I don't think an interface is necessary. Added the javadoc